pkgs/development/python-modules/geopandas/default.nix

```diff
@@ -39,10 +50,30 @@ buildPythonPackage rec {
     shapely
   ];
 
+  optional-dependencies = {
+    all = [
+      psycopg
```

Comment:
Would you please reorder this list alphabetically ? Thanks

Reply:
Ty, done!

Reply:
Thank you.